src/main/scala/thurloe/service/NotificationService.scala

```diff
@@ -0,0 +1,42 @@
+package thurloe.service
+
+import spray.http.MediaTypes._
+import spray.http.StatusCodes
+import spray.routing.HttpService
+import thurloe.service.ApiDataModelsJsonProtocol._
+import thurloe.dataaccess.HttpSendGridDAO
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.{Failure, Success}
+
+trait NotificationService extends HttpService {
+
+  import spray.httpx.SprayJsonSupport.sprayJsonUnmarshaller
+
+  val sendGridDAO = new HttpSendGridDAO
+
+  val postRoute = path("notification") {
+    post {
+      entity(as[Notification]) { notification =>
+        onComplete(sendGridDAO.sendNotification(notification)) {
+          case Success(_) =>
```

Comment:
Can shrink this down to simply `case Success(_) => complete(StatusCodes.OK)` since we don't return a meaningful payload